src/main/java/com/google/cloud/tools/app/api/deploy/DeployConfiguration.java

```diff
@@ -28,17 +28,17 @@
 
   String getDockerBuild();
 
-  boolean isForce();
```

Comment:
It's actually possible this PR is completely unnecesary, we don't set any flags unnecessarily